bitcoinspv/clients/expected_native_client.go

```diff
@@ -0,0 +1,21 @@
+package clients
+
+import (
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/gonative-cc/relayer/bitcoinspv/types"
+)
+
+// NativeClient is interface for interacting with bitcoin-lightclient.
+// Refer to github.com/gonative-cc/bitcoin-lightclient for implementation.
+type NativeClient interface {
```

Comment:
This interface is not used anywhere

Reply:
To be changed in next PR, as per our discussion